Signal/src/AppDelegate.m

```diff
@@ -199,13 +199,22 @@ - (UIViewController *)loadingRootViewController
         [[UIStoryboard storyboardWithName:@"Launch Screen" bundle:nil] instantiateInitialViewController];
 
     BOOL shouldShowUpgradeLabel = NO;
-    NSString *previousVersion = AppVersion.instance.lastCompletedLaunchAppVersion;
+    NSString *lastAppVersion = AppVersion.instance.lastAppVersion;
+    NSString *lastCompletedLaunchAppVersion = AppVersion.instance.lastCompletedLaunchAppVersion;
+    BOOL mayNeedUpgrade = ([TSAccountManager isRegistered] &&
```

Comment:
I'd be more concerned, but it only affects the appearance of the label. Expect an outrageous fury of indignation if we start re-inventing the "one shot" approach to migrations taken in VersionMigrations